src/test/resources/emails/systemCrashReportEmail.html

```diff
@@ -33,7 +33,7 @@
 <pre><code>
 
 java.lang.AssertionError: invalid parameter
-	at teammates.test.cases.logic.EmailGeneratorTest.testSystemCrashReportEmailContent(EmailGeneratorTest.java:123)
+	at teammates.test.cases.logic.EmailGeneratorTest.testSystemCrashReportEmailContent(EmailGeneratorTest.java:137)
```

Comment:
This is because the line number is dependent on where the `AssertionError` is defined in `EmailGeneratorTest`. In this manner this test is fragile, but since we're not changing the emails test that often this should not be a big problem.